pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Its probably worth writing `mkswap` in Go/Rust at some point, I suspect it is pretty trivial. `swapon` is a single system call, so thats easy. I don't know what the crypto setup involves.